.github/CONTRIBUTING.md

```diff
@@ -27,8 +26,11 @@ If you're not used to this workflow with git, you can start with some [docs from
 
 ## Tests
 
-You can optionally test your changes by running the pipeline locally. Then it is recommended to use the `debug` profile to
-receive warnings about process selectors and other debug info. Example: `nextflow run . -profile debug,test,docker --outdir <OUTDIR>`.
+You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile. Execute all the tests with the following command:
```

Comment:
Don't know if we should keep the previous version until nf-test is really implemented at the pipeline level

Reply:
I have open issues for this, so I will do it for the next PR! :)